demos/iperf3/kind/README.md

```diff
@@ -1,231 +1,188 @@
-# <ins>Scenario with iPerf3 Connectivity between 2 clusters<ins>
-In this test we check iPerf3 connectivity between different Kind cluster using the MBG components.  
+# Scenario with iPerf3 Connectivity between 2 clusters
```

Comment:
@kfirtoledo : I think the README can expand beyond kind cluster also. Would be good to mention how to deploy if the deployment is on k8s cluster instead of kind as a separate note.

Reply:
Most of the latter part remains the same. Except the initial part on. how the gw deployment should happen.